doc/whats_new.rst

```diff
@@ -86,6 +86,12 @@ API
   convergence. User can constrain parameter ranges and specify solver,
   by `Carolina Fernandez Pujol`_ in :gh:`652`
 
+- :func:`network.add_tonic_bias` argument `amplitue` now  accepts a 
+  cell_type(str)-amplitude(float) dictionary in case `cell_type` argument is None,
```

Comment:
this is too complicated for a new user to understand ... can you simplify the description? Something like:

"cell-specific tonic bias can now be provided using the argument `amplitude` in network.add_tonic_bias"

Reply:
Done